apps/src/templates/manageStudents/MoveStudents.jsx

```diff
@@ -193,17 +265,72 @@ class MoveStudents extends Component {
         />
         <BaseDialog
           useUpdatedStyles
-          isOpen={this.state.isDialogOpen}
+          isOpen={isDialogOpen}
           style={styles.dialog}
           handleClose={this.closeDialog}
         >
-          <Table.Provider
-            columns={columns}
-            style={styles.table}
-          >
-            <Table.Header />
-            <Table.Body rows={sortedRows} rowKey="id" />
-          </Table.Provider>
+          <div style={styles.container}>
+            <Table.Provider
+              columns={columns}
+              style={styles.table}
+            >
+              <Table.Header />
+              <Table.Body rows={sortedRows} rowKey="id" />
+            </Table.Provider>
+            <div style={styles.rightColumn}>
+              <div>{i18n.selectStudentsToMove()}</div>
+              <label
+                htmlFor="sections"
+                style={styles.label}
+              >
+                {`${i18n.moveToSection()}:`}
+              </label>
+              <select
+                name="sections"
+                style={styles.input}
+                onChange={this.onChangeSection}
+              >
+                {this.renderOptions()}
+              </select>
+              {otherTeacherSelected &&
+                <div>
+                  <label
+                    htmlFor="sectionCode"
+                    style={styles.label}
+                  >
+                    {`${i18n.enterSectionCode()}:`}
+                  </label>
+                  <input
+                    required
+                    name="sectionCode"
+                    style={{...styles.input, width: 225}}
```

Comment:
nit: you'll see it all over the place, but in general we try to avoid inline styles

Reply:
good call! i'll pull that into its own `styles` key